tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -256,6 +256,15 @@ platform_tests/api/test_fan_drawer_fans.py::TestFanDrawerFans::test_set_fans_spe
     conditions:
       - "asic_type in ['mellanox', 'cisco-8000']"
 
+#######################################
+##### api/test_module.py #####
```

Comment:
Can you add some blanks here to keep alignment?

Reply:
One change needs your attention and action: 
Previous Microsoft CLA is replaced by LF EasyCLA (Individual CLA) [https://docs.linuxfoundation.org/lfx/easycla](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.linuxfoundation.org%2Flfx%2Feasycla&data=05%7C01%7CGenHwa.Chiang%40microsoft.com%7Cce525b22ea284e7f66a608da794cd05a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955668076233615%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3XjetZ%2BDHRkvFGyeeTIjzDMv1rVKeyQrnhDQ2wZ1iyI%3D&reserved=0), each contributor needs to sign this ICLA before they can merge PR. No corporation CLA is permitted. The sign process is straightforward.
1.	For existing PR, PR owner can add a comment '/easycla' in PR, it will trigger the EasyCLA check. Then PR owner needs to sign the CLA (individual version) by following the guidance in the EasyCLA check message
2.	For new PR, EasyCLA check will be invoked automatically, PR owner needs to sign the CLA by following the guidance in the EasyCLA check message (see  below sample)
3.	You only need to sign once, then the sign will be validated against all future PRs among all sonic-net repos.

Reply:
/easycla

Reply:
/easycla